hack/lib/constants.sh

```diff
@@ -0,0 +1,150 @@
+  ldflags+=($(os::build::ldflag "${OS_GO_PACKAGE}/pkg/version.commitFromGit" "${OS_GIT_COMMIT}"))
+  ldflags+=($(os::build::ldflag "${OS_GO_PACKAGE}/pkg/version.buildDate" "${buildDate}"))
+
+  # The -ldflags parameter takes a single string, so join the output.
+  echo "${ldflags[*]-}"
+}
+readonly -f os::build::ldflags
+
+# No-op
+function os::build::generate_windows_versioninfo() {
+  :
+}
+readonly -f os::build::generate_windows_versioninfo
+
+function os::build::clean_windows_versioninfo() {
+  :
+}
+readonly -f os::build::clean_windows_versioninfo
+
+# os::util::list_go_src_files lists files we consider part of our project
+# source code, useful for tools that iterate over source to provide vet-
+# ting or linting, etc.
+#
+# Globals:
+#  None
+# Arguments:
+#  None
+# Returns:
+#  None
+function os::util::list_go_src_files() {
+	find . -not \( \
+		\( \
+		-wholename './_output' \
+		-o -wholename './.*' \
+		-o -wholename '*/vendor/*' \
+		\) -prune \
+	\) -name '*.go' | sort -u
+}
+readonly -f os::util::list_go_src_files
+
+# os::util::list_go_src_dirs lists dirs in origin/ and cmd/ dirs excluding
+# doc.go useful for tools that iterate over source to provide vetting or 
+# linting, or for godep-save etc.
+#
+# Globals:
+#  None
+# Arguments:
+#  None
+# Returns:
+#  None
+function os::util::list_go_src_dirs() {
+	os::util::list_go_src_files | cut -d '/' -f 1-2 | grep -v ".go$" | grep -v "^./cmd" | LC_ALL=C sort -u
+	os::util::list_go_src_files | grep "^./cmd/"| cut -d '/' -f 1-3 | grep -v ".go$" | LC_ALL=C sort -u
+}
+readonly -f os::util::list_go_src_dirs
+
+# os::util::list_test_packages_under lists all packages containing Golang test files that we 
+# want to run as unit tests under the given base dir in the source tree
+function os::util::list_test_packages_under() {
+    local basedir=$*
+
+    # we do not quote ${basedir} to allow for multiple arguments to be passed in as well as to allow for
+    # arguments that use expansion, e.g. paths containing brace expansion or wildcards
+    find ${basedir} -not \(                   \
+        \(                                    \
+              -path 'vendor'                  \
+              -o -path '*_output'             \
+              -o -path '*.git'                \
+              -o -path '*vendor/*'            \
+              -o -path '*test/*'              \
+        \) -prune                             \
+    \) -name '*_test.go' | xargs -n1 dirname | sort -u | xargs -n1 printf "${OS_GO_PACKAGE}/%s\n"
+}
+readonly -f os::util::list_test_packages_under
+
+# os::util::list_go_deps outputs the list of dependencies for the project.
+function os::util::list_go_deps() {
+  go list -f '{{.ImportPath}}{{.Imports}}' ./pkg/... ./cmd/... | tr '[]' '  ' | 
+    sed -e 's|${OS_GO_PACKAGE}/vendor/||g'
+}
+
+# OS_ALL_IMAGES is the list of images built by os::build::images.
+readonly OS_ALL_IMAGES=(
+  openshift/origin-pod
```

Comment:
fix this

Reply:
What should this be, ```openshift/k8s.io/heapster```? Does this function that follows also need to be updated?

```bash
function os::build::images() {
  tag_prefix="${OS_IMAGE_PREFIX:-"openshift/origin"}"
  os::build::image "${tag_prefix}-pod" images/pod
}
```

Reply:
`openshift/origin-heapster`, and yes, that also needs to be updated.